index.html

```diff
@@ -214,5 +214,45 @@ <h3>Multi-Post Stories</h3>
 		  </ul>
 		</section>
 	</main>
+
+	<!-- The footer with the form and social links -->
+	<footer class="footer">
+		<div class="ttl-para">
+			<h2 class="contact-us">contact us</h2>
+			<p class="para ftr-para">
+				If you have an application you are interested in developing, 
+				a feature that you need built or a project that needs coding. I’d love to help with it 
+			</p>
+		</div>
+
+		<!-- The form -->
+		<form  action="https://formspree.io/f/mnqlvaya"	method="POST">
+			<div>
+				<label for="name">Name</label>
+				<input type="text" name="name" class="name name-email" id="name" maxlength="30" required placeholder="Enter your name">
+	
+				<label for="email">Email</label>
+				<input type="email" name="email" class="email name-email" id="email" required placeholder="email@email.com">
+			</div>
+
+			<textarea name="message" class="message" maxlength="500" placeholder="Enter text here."></textarea>
+
+			<button type="submit" class="see-project">
+				see project
+			</button>
+		</form>
```

Comment:
- [ ] The code in the index.html file has a different webpage from that you have in the live demo.
Here is your live demo version.
![Screenshot from 2021-08-17 15-46-18](https://user-images.githubusercontent.com/65501535/129728383-6fab117e-7f43-46c8-bf0c-45ae29ed0f14.png)


And here is the webpage based on your code.
![Screenshot from 2021-08-17 15-45-50](https://user-images.githubusercontent.com/65501535/129728556-67be76ed-3e69-438f-8101-d6904d10a66b.png)

- Fix this issue and let's stick to the right version.

Reply:
done, thanks, even this is not a project requierement!